Classes/Controller/BuilderModuleController.php

```diff
@@ -152,6 +153,9 @@ public function initializeAction()
      */
     public function indexAction()
     {
+//        $parserService = new ParserService();
+//        $file = '/app/public/typo3conf/ext/test_extension/Classes/Domain/Model/Main.php';
+//        $parserService->parseFile($file);
```

Comment:
commented out on purpose?

Reply:
Oops - no. That was added while testing...